torchvision/models/googlenet.py

```diff
@@ -56,25 +60,25 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False, ini
 
         self.inception5a = Inception(832, 256, 160, 320, 32, 128, 128)
         self.inception5b = Inception(832, 384, 192, 384, 48, 128, 128)
-        if aux_logits:
-            self.aux1 = InceptionAux(512, num_classes)
-            self.aux2 = InceptionAux(528, num_classes)
+
+        self.aux1 = InceptionAux(512, num_classes)
+        self.aux2 = InceptionAux(528, num_classes)
+
         self.avgpool = nn.AdaptiveAvgPool2d((1, 1))
-        self.dropout = nn.Dropout(0.4)
+        self.dropout = nn.Dropout(0.2)
```

Comment:
I suppose the value of dropout was wrong before? Can you expand on this?

Reply:
The paper states 0.4 while TensorFlow uses 0.2. They probably decreased dropout because they added batch normalization.